types/joint.d.ts

```diff
@@ -615,9 +624,9 @@ export namespace dia {
 
         paper: Paper | null;
 
-        initFlag: CellView.FlagLabel;
+        initFlag(): CellView.FlagLabel;
 
-        presentationAttributes: CellView.PresentationAttributes;
+        presentationAttributes(): CellView.PresentationAttributes
```

Comment:
Looks good, I should have remembered to change `initFLag` and `presentationAttributes`. I think there is a semi-colon missing here. Part of the build process has failed on this PR for some reason.